README.md

```diff
@@ -22,46 +22,74 @@
+|:---------------:|:-----------:|:--------------------------------------------------------------------:|
+|       npm       | &ge; 7.11.x |  Node package manager for resolving/installing project dependencies  |
+|      node       | &ge;14.17.x |                          NodeJS interpreter                          |
+|     docker      |  &ge;18.x   |           Installing/removing/managing containers & images           |
+| docker-compose  | &ge;1.25.x  |       Convenience utility for grouped management of containers       |
+|       jdk       |  17.x       |        For running the integration tests using groovy 2.5.18         |
 
-A simple starter kit to start writing node app with the gov uk front end toolkit.
 
 ## Getting started
 
-The easiest way to get started is to use docker compose to download and run the three required containers.
+Offender-Categorisation is a nodeJS application which by default starts up and listens on URL http://localhost:3000
 
-`docker-compose pull`
+It has services on which it depends :
 
-`docker-compose up`
+|              Dependency               | Description                                            | Default                                                                 | Override Env Var                                                                          |
+|:-------------------------------------:|:-------------------------------------------------------|:------------------------------------------------------------------------|:------------------------------------------------------------------------------------------|
+|              prison-api               | Nomis API providing prisons/offender information       | http://localhost:8080                                                   | ELITE2API_ENDPOINT_URL                                                                    |
+|              hmpps-auth               | OAuth2 API server for authenticating requests          | http://localhost:9090/auth                                              | NOMIS_AUTH_EXTERNAL_URL<br>NOMIS_AUTH_URL                                                 |
+|             risk-profiler             | Risk Profiler service                                  | http://localhost:8082/                                                  | RISK_PROFILER_ENDPOINT_URL                                                                |
+|          allocation-manager           | Allocation manager                                     | http://localhost:8083/                                                  | ALLOCATION_MANAGER_ENDPOINT_URL                                                           |
+|               postgres                | PostgreSQL database server for offender-categorisation | psql://localhost:5432                                                   | None - required locally                                                                   |
```

Comment:
"None - required locally" - let's discuss this

Reply:
Resolved